src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -93,6 +116,44 @@ public ModelAndView synchronizeWithCanvas(@PathVariable String sectionId) throws
         return page;
     }
 
+    @RequestMapping(value = "/{sectionId}/save", params = "pushGradesToCanvas", method = RequestMethod.POST)
+    public ModelAndView pushGradesToCanvas(@PathVariable String sectionId, @ModelAttribute("courseConfigurationForm") @Valid CourseConfigurationForm classSetupForm, BindingResult bindingResult) throws NoLtiSessionException{
+        LOG.info("eid: " + canvasService.getEid() + " is pushing grades for course # " + canvasService.getCourseId() + " to Canvas");
+        ModelAndView page = new ModelAndView("forward:/courseConfiguration/" + sectionId);
+
+        Long courseId = Long.valueOf(canvasService.getCourseId());
+
+        boolean isSimpleAttendance = classSetupForm.getSimpleAttendance();
+        List<AttendanceSummaryModel> summaryForSections = isSimpleAttendance ?
+                reportService.getSimpleAttendanceSummaryReport(Long.parseLong(sectionId)) :
+                reportService.getAviationAttendanceSummaryReport(Long.parseLong(sectionId));
+
+        List<Error> submissionErrors = assignmentSubmitter.submitCourseAttendances(isSimpleAttendance, summaryForSections, courseId, canvasService.getOauthToken(), classSetupForm);
```

Comment:
The value of isSimpleAttendance is contained in the classSetupForm which is also passed to the submitCourseAttendances method. Unless there is a reason to pass the value separately (such as a need for the value to be different than what is in the form in some cases) it would be better to not pass the boolean separately.